core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -52,27 +44,12 @@ case class EntityRecord(id: Long,
                         deletedDate: Option[Timestamp]
 ) extends EntityRecordBase
 
-case class EntityRecordWithInlineAttributes(id: Long,
-                                            name: String,
-                                            entityType: String,
-                                            workspaceId: UUID,
-                                            recordVersion: Long,
-                                            allAttributeValues: Option[String],
-                                            deleted: Boolean,
-                                            deletedDate: Option[Timestamp]
-) extends EntityRecordBase
-
```

Comment:
Deleted this and updated anything that used it to use `EntityRecord` instead (`EntityRecord` is just above; expand the github view to see it)